project.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "dialtone",
+  "targets": {
+    "release-local": {
+      "executor": "nx:run-commands",
+      "outputs": [],
+      "dependsOn": [
+        {
+          "projects": [
+            "dialtone-css",
+            "dialtone-icons",
+            "dialtone-tokens",
+            "dialtone-vue2",
+            "dialtone-vue3",
```

Comment:
Don't do this right now, but maybe we want to make these folders named just "css" "icons" "tokens" "vue2" "vue3" since all of it is now "dialtone". Obviously a pretty annoying change to change folder names of everything but maybe in the future.

Reply:
Yeah, I was thinking about that, but as the packages are actually named `@dialpad/dialtone-*` I left it like that, to be consistent with the naming